app/courses/page.tsx

```diff
@@ -57,34 +70,38 @@ const Path = ({
   return (
     <Link href={isFree ? `/courses/${slug}` : `/signup`}>
       <Card
+        variant={isFree ? "whiteShadow" : "locked"}
         position={"relative"}
         flexDir={"row"}
         justifyContent={"space-between"}
       >
-        {!isFree && (
-          <Flex
-            zIndex={50}
-            className="bg-gray-100/60"
-            position={"absolute"}
-            top={0}
-            bottom={0}
-            left={0}
-            right={0}
-            justifyContent={"center"}
-            alignItems={"center"}
-          >
-            <FaLock size={60} className="text-gray-800/50" />
-          </Flex>
-        )}
         <Flex flexDir={"column"} gap={1}>
-          <Tag
-            w={"fit-content"}
-            size={"sm"}
-            colorScheme="hollywood"
-            letterSpacing={1}
-          >
-            {level}
-          </Tag>
+          <Flex justifyContent={"space-between"} alignItems={"center"}>
+            <Tag
+              w={"fit-content"}
+              size={"sm"}
+              bg="#FDB548"
+              letterSpacing={1}
+              mr={2}
+            >
+              {level}
+            </Tag>
+            {!isFree && (
+              <Tag
+                w={"fit-content"}
+                size={"sm"}
+                colorScheme="hollywood"
+                letterSpacing={1}
+                gap={2}
+                bg="#EA0091"
```

Comment:
Setting a `colorScheme` and then a `bg` is not advised, create a new `colorScheme` that what you want.